### PR TITLE
Add ETH test networks and fix spelling mistake

### PR DESCRIPTION
**Pull request by 0xCLARITY**
Add `btc-testnet` header and switch `eth-testnet-name` header for some real ETH testnet names.

### CODE REVIEW

File: dist/index.php [hunk 1]
```diff
@@ -146,7 +146,7 @@
                         </div>
                     </div>
 
-                    <?php if (!$payIDValidator->hasValidationOccured()) : ?>
+                    <?php if (!$payIDValidator->hasValidationOccurred()) : ?>
```
Comment (0xCLARITY) [inline dist/index.php:149]: Fix spelling of `occured => occurred`, here and everywhere else.

File: dist/lib/PayIDValidator.php [hunk 2]
```diff
@@ -5,11 +5,18 @@ class PayIDValidator {
     /**
      * Supported networks
      */
-    const NETWORK_ACH = 'ach';
     const NETWORK_ALL = 'all';
+
+    const NETWORK_ACH = 'ach';
```
Comment (0xCLARITY) [inline dist/lib/PayIDValidator.php:10]: I saw these were alphabetical, but I think `all` is a special value different from an actual network, so I put it first.

File: dist/lib/PayIDValidator.php [hunk 3]
```diff
@@ -5,11 +5,18 @@ class PayIDValidator {
...
+
     const NETWORK_BTC_MAINNET = 'btc-mainnet';
+    const NETWORK_BTC_TESTNET = 'btc-testnet';
```
Comment (0xCLARITY) [inline dist/lib/PayIDValidator.php:13]: Added BTC testnet as a header.

File: dist/lib/PayIDValidator.php [hunk 4]
```diff
@@ -5,11 +5,18 @@ class PayIDValidator {
...
+
     const NETWORK_ETH_MAINNET = 'eth-mainnet';
-    const NETWORK_ETH_TESTNET = 'eth-testnet-name';
+    const NETWORK_ETH_ROPSTEN = 'eth-ropsten';
+    const NETWORK_ETH_KOVAN = 'eth-kovan';
+    const NETWORK_ETH_RINKEBY = 'eth-rinkeby';
```
Comment (0xCLARITY) [inline dist/lib/PayIDValidator.php:18]: Switch `eth-testnet-name` to some actual ETH testnet headers.

**Comment by 0xCLARITY** [comment]
@rswarthout , not sure how to tag you as a reviewer on this, but feel free to take a look.

File: dist/index.php (hunk 1 above)
Comment (rswarthout) [reply dist/index.php:149]: Spelling can be my achilles heel sometimes. :smile:

File: dist/lib/PayIDValidator.php (hunk 2 above)
Comment (rswarthout) [reply dist/lib/PayIDValidator.php:10]: Works for me.

**Comment by rswarthout** [comment]
I appreciate these changes.

**Pull request merged**